### ISSUE TITLE
Web Player [56h]

### ISSUE THREAD

**Issue by karina4now** [TRASH-INC/trash-inc.github.io#84]
Description
Currently we have a very basic and alpha implementation of a web player so that users can share their videos on the web and view them. We need to upgrade this experience and part of this is a launch blocker for the TFA release early May.

Current experience:https://video.trash.app/player/f09c9a7b-0af0-4aa6-9288-5ab1374bd79e
Objective / outcome
Upgrade the current experience so that it is on par with TikTok, Instagram, Byte. Additionally:

Upgrade the current web view experience for a single video, with the consideration that user will be coming from one of thepreview links here, and the metadata presented should reflect that (a lot of copy inspo in that doc).

Offer support for channels, so that a user can watch multiple videos on a page. Channels includes:

The channels in the app (discover screen, for example “GOAT). This is a launch blocker for the TFA launch

The user profiles, which are also a “channel” of user videos
Hypothesis/ key results
Increased shares

Increased views
Acceptance criteria and must have scope
Upgrade current experience:

Layout should account for horizontal or vertical video within a responsive website

Video player should have a better affordance for play button, and clean up the player UI in general (second part NTH)

Update metadata associated with the video based on the preview link copy

Update call to action to meet our growth key result

Once support for channels is offered, add link support to the user’s profile (channel) and a “watch more…” link to GOAT.

Support for channels:

Video player needs to support multiple videos with a next and previous button

All the things listed in part 1 above

Potentially a CTA to watch more channels, like a grid of others to explore (NTH)
Nice to have or exploratory scope
Better player UI beyond putting a nice big rainbow play button on it

Contextual links to more videos ie. profiles, channels
Stakeholders
Han, Armand
How long and Risks
A basic version of part 2 needs to be accomplished before we can hard launch TFA, so the timeline is 2 weeks from the beginning of Sprint 6 (4/28)
